index.js

```diff
@@ -23,6 +23,29 @@ module.exports = function (sails) {
             migrate = sails.config.models.migrate;
             sails.log.verbose('Migration: ' + migrate);
 
+            //potentially enable query logging at different logging levels
+            if (connection.options.loglevel) {
+                if (connection.options.loglevel === 'verbose') {
+                    connection.options.logging = sails.log.verbose;
+                } else if (connection.options.loglevel === 'debug') {
+                    connection.options.logging = sails.log.debug;
+                }
+            } else {
+                connection.options.logging = null;
+            }
+
+            //support same environment variables as waterline
+            if (!!process.env.LOG_QUERIES) {
+                connection.options.logging = sails.log.debug;
+            }
+
+            if (typeof connection.options.namespace !== 'undefined' && connection.options.namespace) {
+                sails.log.verbose("CLS is enabled in the "+connection.options.namespace+" namespace");
+                global.Sequelize.cls = require('continuation-local-storage').createNamespace(connection.options.namespace);
```

Comment:
Some comments:
* Take a look at 3rd line. There is a cls initialization there. So this check should be there.
* Also new versions of sequelize prohibit direct use of cls:
    sequelize deprecated Sequelize.cls should not be set directly. Use Sequelize.useCLS()